src/implementations/twig/components/blockquote/blockquote.story.js

```diff
@@ -1,28 +1,61 @@
-import { withKnobs, text } from '@storybook/addon-knobs';
 import { withNotes } from '@ecl/storybook-addon-notes';
 import withCode from '@ecl/storybook-addon-code';
-import { getExtraKnobs, tabLabels } from '@ecl/story-utils';
 
 import defaultData from '@ecl/specs-component-blockquote/demo/data';
 import blockquote from './blockquote.html.twig';
 import notes from './README.md';
 
-const prepareQuote = (data) => {
-  data.citation = text('citation', data.citation, tabLabels.required);
-
-  data.author = text('author', data.author, tabLabels.required);
+const getArgTypes = () => {
+  const argTypes = {};
+  argTypes.author = {
+    name: 'Author',
+    type: { name: 'string', required: true },
+    description: 'Author of the citation',
+    table: {
+      type: { summary: 'string' },
+      defaultValue: { summary: '' },
+      category: 'Content',
+    },
+    control: {
+      type: 'text',
+    },
+  };
+  argTypes.citation = {
```

Comment:
a small detail, but can we put the citation before the author in the control list?
That would reflect the normal reading order of the component